tests/monotouch-test/System.Net.Http/MessageHandlers.cs

```diff
@@ -110,5 +112,45 @@ public void TestNSUrlSessionHandlerCookies ()
 		}
 #endif
 
+		// ensure that if we have a redirect, we do not have the auth headers in the following requests
+#if !__WATCHOS__
+		[TestCase (typeof (HttpClientHandler))]
+		[TestCase (typeof (CFNetworkHandler))]
+#endif
+		[TestCase (typeof (NSUrlSessionHandler))]
+		public void RedirectionWithAuthorizationHeaders (Type handlerType)
+		{
+			PrintHandlerToTest ();
```

Comment:
that line does not seems helpful - it just print all types, every time it's executed